CONTRIBUTING.md

```diff
@@ -1,219 +1,193 @@
-
-* BallerinaLexer.g4 - Contains the lexer rules for Ballerina grammar. Lexer is responsible for tokenizing an input Ballerina source code.
-* BallerinaParser.g4 - Contains the parser rules. Parser listens to the token stream generated by the lexer. High level grammar productions/abstractions are defined in the parser using those tokens.
-
-> TIP: If you want to check and validate a grammar rule you just wrote, you can use the antlr4 plugin of your IDE. See Setting up your development environment section for more details.
-
-#### Generating Parsers
-
-Once a change is done to the any of the grammar files, it is needed to re-generate the parsers. To generate parsers, we first need to download `antlr-complete-4.5.3.jar` file. Then navigate to `<ballerina>/docs/grammar`, and generate the lexer followed by the parser (order is important) using the following command:
-
-```
-java -jar /<path-to-antlr-jar>/antlr-4.5.3-complete.jar *.g4 -package org.wso2.ballerinalang.compiler.parser.antlr4 -o ../../compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/.
-```
-
-The above command will autogenerate some Java classes. Out of the autogenerated classes, `BallerinaParserBaseListener.java` in particular is important, as the Ballerina AST builder is written on top of it. Thus, if any new rules are added to the BallerinaParser.g4, above command will generate new methods in the `BallerinaParserBaseListener.java`, and you need to override those newly added methods inside `BLangParserListener.java` accordingly.
-
-### Ballerina Compiler 
-https://medium.com/@sameera.jayasoma/ballerina-compiler-design-3406acc2476c
-
-### Ballerina VM
-https://medium.com/@sameera.jayasoma/ballerina-runtime-evolution-f82305e4ab8e
-
-### Creating a Patch for Review
-
-#### Fork the Repo
-#### Make Your Changes
-#### Add Unit Tests
-
-Build process automatically runs all the tests.
-
-#### Commit to Your Fork
-
-We follow these commit message requirements:
-
-* Separate subject from body with a blank line
-* Limit the subject line to 50 characters
-* Capitalize the subject line
-* Do not end the subject line with a period
-* Use the imperative mood in the subject line
-* Wrap the body at 72 characters
-* Use the body to explain what and why vs. how
-
-Please find details at: [https://chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-
-## Proposing Changes To Ballerina
-
-Start with the discussion in the Ballerina-Dev Google Group.
-
-Once there is enough consensus around the proposal, you will likely be asked to file an Issue in GitHub and label it as Proposal, to continue the discussion on details there.
-
-
+# Contributing to Ballerina
+
+Ballerina is a cloud native programming language and runtime, including a message broker and API gateway, which makes it easy to create resilient services that integrate and orchestrate transactions across distributed endpoints. 
```

Comment:
I think this "cloud native programming language" needs to be updated with the new messaging as well. Check and see

Reply:
And we don't have an API gateway which ships with Ballerina itself AFAIK

Reply:
We don't have a message broker too now

Reply:
Shall we use "Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software that just works" ? This is the new description used in https://stagedev.ballerina.io/